extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -320,13 +322,92 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
         });
     }
 
+    _handleDeleteOp(entry) {
+        if (entry.type !== 'delete' ||
```

Comment:
As this is tested before calling the function, [here](https://github.com/scality/backbeat/pull/2432/files#diff-37c9ecda10c884ef79419b6744d2491598248b6511813899f07af1909cbf2f43R406) do we really need to check it again ?

Reply:
I see that you made the change but you also removed this test, is it intended ?
```            
entry.key.startsWith(mpuBucketPrefix)) {
```

Reply:
With the switch to the oplog populator and change streams, this check is no longer needed as we don't listen to changes on the shadow buckets anymore